src/Common/src/Interop/Unix/libssl/Interop.libssl.msbuild

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup>
```

Comment:
@stephentoub / @ellismg thoughts on the .msbuild sources-include list?  I think I'm mildly against, but I can't articulate why (something like "but are their needs really *identitical*?")